bin/varnishncsa/varnishncsa.c

```diff
@@ -1017,44 +1041,44 @@ dispatch_f(struct VSL_data *vsl, struct VSL_transaction * const pt[],
 				skip = 1;
 				break;
 			case SLT_PipeAcct:
-				frag_fields(0, b, e,
+				frag_fields(FMTPOL_FIRST, b, e,
 				    3, &CTX.frag[F_I],
 				    4, &CTX.frag[F_O],
 				    0, NULL);
 				break;
 			case SLT_BackendOpen:
-				frag_fields(1, b, e,
+				frag_fields(FMTPOL_ALWAYS, b, e,
 				    3, &CTX.frag[F_h],
 				    0, NULL);
 				break;
 			case SLT_ReqStart:
-				frag_fields(0, b, e,
+				frag_fields(FMTPOL_FIRST, b, e,
 				    1, &CTX.frag[F_h],
 				    0, NULL);
 				break;
 			case SLT_BereqMethod:
 			case SLT_ReqMethod:
-				frag_line(0, b, e, &CTX.frag[F_m]);
+				frag_line(FMTPOL_REQ_SIDE(), b, e, &CTX.frag[F_m]);
 				break;
```

Comment:
My suggestion implies that we could no longer share the same blocks for Req/Bereq switch cases, respectively Resp/Beresp.